mesheryctl/internal/cli/root/system/testdata/reset/TestResetContexts.yaml

```diff
@@ -6,7 +6,7 @@ contexts:
     components:
       - meshery-istio
     channel: stable
-    version: v0.5.56
+    version: v0.66.6
```

Comment:
Do you mean `v0.6.6` here and all the related files below?

Reply:
I reverted it back to the original.